sdcm/utils/benchmarks.py

```diff
@@ -150,13 +152,21 @@ def _get_db_node_instance_type(self):
 
     def _install_ubuntu_prerequisites(self) -> bool:
         package_list = ["make", "automake", "libtool", "pkg-config", "libaio-dev", "fio"]
+        time_1 = time.time()
```

Comment:
Please, name it more self-explanatory [1].

[1] https://docs.google.com/document/d/1jihgOKb5iGRlD8_HQ92O0JbLk1kASUoZT23i_MXFSKI/edit#heading=h.6byizauykwn9

Reply:
Fixed with rebase